src/components/AsideHeader/components/CompositeBar/Item/Item.module.scss

```
@@ -116,6 +128,48 @@ $actionLeftRightMargin: 10px;
+            visibility 0s linear 0.15s,
+            color 0.15s ease;
+
+        &_pinned {
+            color: var(--g-color-text-secondary);
+        }
+
+        &:hover {
+            color: var(--g-color-text-primary);
+        }
+
+        :global(.g-button__icon) {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)